model/articles.model.js

```diff
@@ -38,3 +38,42 @@ exports.checkArticleExist = (article_id)=>{
     })
 }
 
+exports.updateVote = (article_id, inc_votes) => {
+    if (inc_votes === 0 || typeof inc_votes !== 'number') {
+      return Promise.reject({ status: 400, msg: 'Bad request' });
+    }
```

Comment:
This is a good custom error handler, but I inc_votes being 0 isn't what we'd usually call a bad request, so I recommend removing it